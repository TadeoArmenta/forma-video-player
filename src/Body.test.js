import React, {Component} from 'react'
import ShallowTestRenderer from 'react-test-renderer/shallow'

import {Body} from './Body'

describe('Body', () => {

  class TestComponent {render() {return null}}

  let renderer

  beforeEach(() => {
    renderer = new ShallowTestRenderer()
  })

  it('should render layout', () => {
    const layout = {
      Left: <TestComponent />,
      Middle: <TestComponent />,
      Right: <TestComponent />,
    }
    const rendered = renderer.render(<Body layout={layout} styles={{}} />)

    expect(rendered).toMatchSnapshot()
  })

})
