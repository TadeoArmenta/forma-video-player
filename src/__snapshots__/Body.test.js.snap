// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body should render layout 1`] = `
<View
  style={undefined}
>
  <TestComponent
    defaultStyles={
      Object {
        "justifyContent": "flex-start",
      }
    }
  />
  <TestComponent
    defaultStyles={
      Object {
        "justifyContent": "center",
      }
    }
  />
  <TestComponent
    defaultStyles={
      Object {
        "justifyContent": "flex-end",
      }
    }
  />
</View>
`;
